### PR TITLE
fix/Texfield_bug

### PR DESCRIPTION
**Pull request by williamricardoardila**

### CODE REVIEW

File: EmeraldIOS/Components/SwiftUi/TextField/EmeraldSwiftUiTextField.swift [hunk 1]
```diff
@@ -186,7 +186,7 @@ struct EmeraldSwiftUiTextField_Previews: PreviewProvider {
         var body: some View {
             EmeraldSwiftUiTextField(
                 textFieldType: .normal,
-                text: _name,
+                text: .constant(name),
```
Comment (ronalfabra) [inline EmeraldIOS/Components/SwiftUi/TextField/EmeraldSwiftUiTextField.swift:189]: should be $name

File: EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldSwiftUiTextView.swift [hunk 2]
```diff
@@ -95,7 +95,7 @@ struct EmeraldSwiftUiTextView_Previews: PreviewProvider {
         
         var body: some View {
             EmeraldSwiftUiTextView(
-                text: _name,
+                text: .constant(name),
```
Comment (ronalfabra) [inline EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldSwiftUiTextView.swift:98]: should be $name

Comment (williamricardoardila) [reply EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldSwiftUiTextView.swift]: done

File: EmeraldIOS/Components/SwiftUi/TextField/EmeraldSwiftUiTextField.swift (hunk 1 above)
Comment (williamricardoardila) [reply EmeraldIOS/Components/SwiftUi/TextField/EmeraldSwiftUiTextField.swift]: done

**Review by ronalfabra — APPROVED** [review]

**Review by ronalfabra — APPROVED** [review]

**Pull request merged**